### PR TITLE
Improve UI for search results

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
for: https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/498

Position sticky for SearchQueryWrapper on Desktop
Add <kbd>close</kbd> button for search results on mobile

### CODE REVIEW

File: src/app/content/components/Toolbar/index.tsx [hunk 1]
```diff
@@ -161,6 +178,7 @@ export default connect(
   }),
   (dispatch: Dispatch) => ({
     clearSearch: flow(clearSearch, dispatch),
+    closeSearch: flow(closeSearchResultsMobile, dispatch),
```
Comment (janKozie1) [inline src/app/content/components/Toolbar/index.tsx]: Maybe `closeSearchResults` to match `openSearchResults` ?

**Review by janKozie1 — COMMENTED** [review]
The design shows `close` only closing the results, but at least for me `close` means to close the whole search component, not close only the results but keep the input. Not sure what's the proper behavior.

**Comment by PiotrKozlowski** [comment]
> The design shows `close` only closing the results, but at least for me `close` means to close the whole search component, not close only the results but keep the input. Not sure what's the proper behavior.

Lets wait for decision: https://openstax.slack.com/archives/C8V1XET1R/p1580218946129200

**Review by TomWoodward — APPROVED** [review]
passing this to QA

**Review by TomWoodward — APPROVED** [review]

**Pull request merged**